.github/workflows/main.yml

```diff
@@ -31,28 +32,55 @@ defaults:
     shell: bash -el {0}
 jobs:
   tests:
-    name: ${{ matrix.os }}, Python ${{ matrix.python-version }}, ${{ matrix.micromamba && 'micromamba' || 'conda-standalone' }}
+    name: ${{ matrix.os }}, Python ${{ matrix.python-version }}, ${{ matrix.conda-standalone }}
     runs-on: ${{ matrix.os }}-latest
     strategy:
       fail-fast: false
       matrix:
-        os: [macos, ubuntu, windows]
-        python-version: ["3.8", "3.9", "3.10", "3.11"]
-        check-docs: [false]
         include:
+          # UBUNTU
+          - os: ubuntu
+            python-version: "3.8"
+            conda-standalone: conda-standalone
+            check-docs: true
           - os: ubuntu
             python-version: "3.9"
-            micromamba: true
+            conda-standalone: conda-standalone
             check-docs: true
+          - os: ubuntu
+            python-version: "3.10"
+            conda-standalone: conda-standalone-nightly
+          - os: ubuntu
+            python-version: "3.11"
+            conda-standalone: micromamba
+          # MACOS
```

Comment:
are you intentionally skipping Python 3.8 for macOS?

Reply:
Accidental at first, then I thought, hm why not? macOS runners are scarce after all. Then I asked in the call whether we can drop or not for `conda/conda`. I guess I'll add it back?

Reply:
In conda-build we opted to only test the lower and upper versions on macOS. My thought behind that was we'd catch stdlib deprecations with the upper version and we'd catch usage of newer stdlib features that the lower version doesn't support.

Dunno how relevant that would be for `constructor`, just wanted to be sure the exclusion was intentional. 👍🏻

Reply:
I like that. Will apply changes and ping you again. Thanks!

Reply:
Done @kenodegard. Can you reapprove? Thanks 🙏